helix-term/src/commands/typed.rs

```diff
@@ -1473,7 +1474,9 @@ fn tree_sitter_subtree(
             .root_node()
             .descendant_for_byte_range(from, to)
         {
-            let contents = format!("```tsq\n{}\n```", selected_node.to_sexp());
+            let mut contents = String::from("```tsq\n");
+            pretty_print_tree(&mut contents, selected_node)?;
```

Comment:
Very tiny nit but I'd prefer this be fully qualified as `helix_core::syntax::pretty_print_tree(&mut contenst, selected_node)?` since we're unlikely to re-use the function elsewhere in this file

Reply:
Makes sense, just pushed a commit :)